pcsx2/CDVD/CDVD.cpp

```diff
@@ -958,7 +958,7 @@ u8 monthmap[13] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
 
 void cdvdVsync() {
 	cdvd.RTCcount++;
-	if (cdvd.RTCcount < ((gsRegionMode == Region_NTSC) ? 60 : 50)) return;
+	if (cdvd.RTCcount < ((gsVideoMode == NTSC) ? 60 : 50)) return;
```

Comment:
It basically counts the number of vsyncs, once it reaches a "second" worth of vsyncs it adds 1 to the seconds, then resets and counts again, it's simple, neither @gigaherz or @gregory38 were required :P